### DISCUSSION TITLE
Add the ability to find Hidden models

### DISCUSSION THREAD

**Discussion by sownheard** [civitai/civitai#95] [Ideas]
While trying to test the hide models, 
i removed a model. 
Because i dont know the name now i cant find it and unhide it.

**Comment by manuelurenah** [comment]
@sownheard There's a PR https://github.com/civitai/civitai/pull/97 which brings the changes to let you see the list of creators you have marked to hide from your feed. From there you will be able to unhide them if you want. Right now is in the review stage and will be released shortly. Look forward to it!

**Comment by manuelurenah** [comment]
Hey @sownheard ! This has been released already. You should be able to find your follower and hidden list from your user menu by clicking your avatar at the top right corner

![SCR-20221220-et1](https://user-images.githubusercontent.com/12631159/208693152-4ba7c82f-c36d-42f3-a898-88bff93e6fe2.png)

Then selecting the Hidden tab in the following page

![SCR-20221220-et9](https://user-images.githubusercontent.com/12631159/208693221-20b197b5-86bc-4d6e-9e35-6acbc125c43f.png)

Hope this helps!

**Reply by nabileisa** [reply]
Hi @manuelurenah , Although this helps with finding creators a user may have hidden, it doesn't address individual models , If I hide a specific model by mistake I cannot find a way to locate that model again and restore it

**Reply by manuelurenah** [reply]
Hey @nabileisa! We know about it, and it's already in our TODO board. We should have it ready in our next sprint. Thank you for letting us know

**Comment by JustMaier** [comment]
We don't have a link to it right now, but you can view your hidden models here:
[https://civitai.com/?hidden=true](https://civitai.com/?hidden=true)

**Comment by sownheard** [comment]
thank you for your work and effort.

Op zo 5 feb 2023 om 20:57 schreef Justin Maier ***@***.***>:
